src/utilities/getInitialRoute.browser.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, test } from 'vitest'
+import { createRouter } from '@/utilities/createRouter'
+import { component } from '@/utilities/testHelpers'
+
+describe('getInitialRoute', () => {
```

Comment:
Do you think every test should be wrapped in a describe? Vitest automatically groups tests by their file so IMO wrapping all the tests in a describe with the name of the utility being tested (which is the same as the name of the file) is not needed and we should reserve describe for organizing larger test files where we want to group tests of similar nature. If every test is in a single describe we don't need it IMO.